### PR TITLE
Console - add model and webservices for delegation

### PR DESCRIPTION
**Pull request by Vampouille**
This PR create basic webservices to handle delegations. We still need to implements authorization and filtering results.

### CODE REVIEW

File: console/src/main/java/org/georchestra/console/ws/backoffice/delegations/DelegationController.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+	@RequestMapping(value=REQUEST_MAPPING + "/{uid}", method= RequestMethod.GET, produces = "application/json; charset=utf-8")
+	@ResponseBody
+	public String findByUid(@PathVariable String uid) throws JSONException {
+
+		// TODO test if uid correspond to connected user if request came from delegated admin
+		DelegationEntry delegation = this.delegationDao.findOne(uid);
+
+		JSONObject res = new JSONObject();
+		res.put("orgs", delegation.getOrgs());
```
Comment (fvanderbiest) [inline console/src/main/java/org/georchestra/console/ws/backoffice/delegations/DelegationController.java]: also set `uid` field in returned object ?

**Comment by fvanderbiest** [comment]
LGTM otherwise

File: console/src/main/java/org/georchestra/console/ws/backoffice/delegations/DelegationController.java (hunk 1 above)
Comment (Vampouille) [reply console/src/main/java/org/georchestra/console/ws/backoffice/delegations/DelegationController.java]: Done with 2f9b7bd

**Comment by tonio** [comment]
For consistency with other services, I'd have mapped URI like:
 - `/private/delegations` for the list
 - `/private/delegations/:id` to get/update one.

Thanks (-:

**Comment by tonio** [comment]
And return list as an array like for roles, please! Thank you!

**Pull request merged**